web/app.js

```diff
@@ -3077,25 +3037,6 @@ function webViewerKeyDown(evt) {
     }
   }
 
-  // if (typeof PDFJSDev === "undefined" || PDFJSDev.test("GENERIC || CHROME")) {
-  //   // CTRL or META without shift
-  //   if (cmd === 1 || cmd === 8) {
-  //     switch (evt.keyCode) {
-  //       case 83: // s
-  //         eventBus.dispatch("download", { source: window });
-  //         handled = true;
-  //         break;
-
-  //       case 79: // o
-  //         if (typeof PDFJSDev === "undefined" || PDFJSDev.test("GENERIC")) {
-  //           eventBus.dispatch("openfile", { source: window });
-  //           handled = true;
-  //         }
-  //         break;
-  //     }
-  //   }
-  // }
-
```

Comment:
This was [commented and left commented](https://github.com/creativecirclemedia/pdf.js/commit/8954685483134e60fd2ed136af30505877502781), so I just removed it.